### PR TITLE
fix(1319): add getOrgPermissions function

### PR DESCRIPTION
**Pull request by minz1027**
- add a function to check user's permission on an smc organization

Blocked by: https://github.com/screwdriver-cd/data-schema/pull/291

### CODE REVIEW

**Comment by tkyi** [comment]
Need to update README

File: README.md [hunk 1]
```diff
@@ -246,6 +246,30 @@ Permissions for a given token on a repository in the form of:
 1. Resolve with a permissions object for the repository
 2. Reject if not able to get permissions
 
+### getOrgermissions
```
Comment (tkyi) [inline README.md]: missing "P"

```suggestion
### getOrgPermissions
```

File: README.md [hunk 2]
```diff
@@ -246,6 +246,30 @@ Permissions for a given token on a repository in the form of:
...
+| config.username | String | The user to get permissions on (ex: `foo`) |
+| config.token | String | Access token for scm |
+| config.scmContext | String | (optional) The name of scm context |
+
+#### Expected Outcome
+Permissions for a given user on a organization in the form of:
+```js
+{
+    admin: false,
+    member: true,
```
Comment (tkyi) [inline README.md]: dangling comma

Comment (minz1027) [reply README.md]: addressed 👍

**Comment by tkyi** [comment]
Need to add `_getOrgPermissions` to https://github.com/screwdriver-cd/scm-base/blob/master/README.md#extending in the README too

**Pull request merged**